public/src/ui/hud.js

```
@@ -0,0 +1,437 @@
+      if (currentStage) {
+        let statusText = localization.getText('stages.unknownCondition');
+        switch (currentStage.clearCondition.type) {
+          case StageClearConditions.KILL_ALL_ENEMIES:
+            const aliveEnemies = this.game.enemies.filter(
+              (e) => !e.isDead
+            ).length;
+            statusText =
+              aliveEnemies > 0
+                ? localization
+                    .getText('stages.enemiesRemaining')
+                    .replace('{0}', aliveEnemies)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)